docs/users.md

```diff
@@ -11,44 +11,42 @@ As of current knowledge, fractureiser specifically tagets Linux and Windows inst
 ### Windows Instructions
 
 * Open your Start menu with the Windows Key, and type `%localappdata%` - it should appear as such:
-![](media/localappdata.png)
+![search results](media/localappdata.png)
 
-* Inside the Local appdata folder, you must ensure that your Explorer is set to view both `Hidden Items`, and `Protected Operating System Files`. 
- * This can be done from View > Options
- * If you are unsure how to do this, a video explanation [can be found here](https://youtu.be/KLTlTlnXeKs).
- * There is a dectection tool by eitambelahousky at https://github.com/overwolf/detection-tool to automate this 
+* Inside the Local appdata folder, you must ensure that your Explorer is set to view both `Hidden Items`, and `Protected Operating System Files`.
+* This can be done from View > Options
+* If you are unsure how to do this, a video explanation [can be found here](https://youtu.be/KLTlTlnXeKs).
+* There is a dectection tool by eitambelahousky at https://github.com/overwolf/detection-tool to automate this 
 
 * Locate a folder named `Microsoft Edge`. The SPACE between "Microsoft" and "Edge" is important - as `MicrosoftEdge` is a legitimate folder that is used by Edge. The virus simply named it like that to disguise itself.
 * If `Microsoft Edge` is present, you were infected. If this is the case, permanently delete the folder and everything inside it.
- * If the folder can not be deleted, you must stop any Java programs currently running via your Task Manager.
+* If the folder can not be deleted, you must stop any Java programs currently running via your Task Manager.
 
 Afterwards, follow onto the next section of this page that is relevant to you.
 
 ### Linux Instructions
 
-Firstly, ensure whichever method you are using to list files has the ability to view hidden files. If doing this on a terminal, use `ls -a` in the respective directories.
+To begin with, make sure that the method you are using to list files has the capability to display hidden files. If you are using the terminal, use the command `ls -a` in the respective directories.
+
+If any of the following files exist, it indicates a potential infection. In such a case, delete all of them:
 
-If any of the following files exist, you were infected. If this is the case, delete all of them:
 * `~/.config/systemd/user/systemd-utility.service`
 * `/etc/systemd/system/systemd-utility.service`
 * `~/.config/.data/lib.jar`
 
-Upon doing so, if applicable, check your `systemctl` for any changes you may not recognize. Afterwards, follow onto the next section of this page that is relevant to you.
-
-## MacOS Information
-
-The malware does not seem to affect MacOS, so you should be fine. *Recheck this doc in the future if this changes*
+After deleting these files, if applicable, check your `systemctl` for any changes you do not recognize. Then, proceed to the relevant section below based on your situation.
 
-### Scripts
+### Automation Script
```

Comment:
```suggestion
### Scripts
```
I'm not knowledgeable but I don't think this level of verbosity is needed, just "Scripts" should be fine and professional enough, in fact this makes it awkward, and slightly incorrect as there are multiple scripts.